src/rpc_server.rs

```diff
@@ -744,6 +748,14 @@ impl RPC for NeptuneRPCServer {
             .get_all_own_coins_with_possible_timelocks()
             .await
     }
+
+    async fn cpu_temp(self, _context: tarpc::context::Context) -> Option<f64> {
```

Comment:
nit: no need for fully qualified path on Context.

```suggestion
    async fn cpu_temp(self, _context: Context) -> Option<f64> {
```